CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ project(cesium-native
 
 option(PRIVATE_CESIUM_SQLITE "ON to rename SQLite symbols to cesium_sqlite3_* so they won't conflict with other SQLite implemenentations" OFF)
 option(CESIUM_TRACING_ENABLED "Whether to enable the Cesium performance tracing framework (CESIUM_TRACE_* macros)." OFF)
+option(CESIUM_COVERAGE_ENABLED "Whether to enable code coverage" ON)
```

Comment:
the options maybe need to turn off since some compiler flags are added in the release build

Reply:
ok, I changed it to OFF and that will also fix CI since the script doesn't install gcovr